pkg/helm/status.go

```diff
@@ -0,0 +1,56 @@
+package helm
+
+import (
+	"fmt"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+// StatusStep represents the structure of an Status action
+type StatusStep struct {
+	Description string          `yaml:"description"`
+	Arguments   StatusArguments `yaml:"helm"`
+}
+
+// StatusArguments are the arguments available for the Status action
+type StatusArguments struct {
+	Releases []string `yaml:"releases"`
+}
+
+// Status reports the status for a provided set of Helm releases
+func (m *Mixin) Status() error {
+	payload, err := m.getPayloadData()
+	if err != nil {
+		return err
+	}
+
+	var step StatusStep
+	err = yaml.Unmarshal(payload, &step)
+	if err != nil {
+		return err
+	}
+
+	cmd := m.NewCommand("helm", "status")
+
+	for _, release := range step.Arguments.Releases {
+		statusCmd := cmd
+		statusCmd.Args = append(statusCmd.Args, release)
+		statusCmd.Stdout = m.Out
+		statusCmd.Stderr = m.Err
+
+		prettyCmd := fmt.Sprintf("%s %s", statusCmd.Path, strings.Join(statusCmd.Args, " "))
```

Comment:
Does helm status have an alternate form that gives the information in a structured format (such as json)?

In general, I'm trying to make sure that every command has 2 outputs (human readable and something scriptable). So if there's an easy way to tell helm, "hey output this in json", then I suggest we include that in this command. 

Checkout my PR for listing mixins for an example of what that should look like if we can do that. If not, no need to worry about it. Just something to keep in mind for new commands as we add them!